.github/pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+### Type of Change
+
+- [ ] Bug fix (non-breaking change that fixes an issue)
+- [ ] New feature (non-breaking change that adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
```

Comment:
@TooTallNate What do you think about using labels instead of checkboxes here?

- `semver-major`
- `semver-minor`
- `semver-patch`

Reply:
The problem with labels is that issue creators can not set labels, only repo committers/triagers can.

Reply:
Indeed but often times contributors don’t know the semver impact so it might be better to let the maintainers select the label instead

Reply:
SGTM

Reply:
@TooTallNate I updated kodiak in 0f6e28488cd8cd3565a6dded59127133714a562b

Related to https://github.com/chdsbd/kodiak/issues/521